internal/connect/telemetry.go

```diff
@@ -0,0 +1,32 @@
+package connect
+
+import (
+	"log/slog"
+
+	"github.com/SUSE/telemetry"
+)
+
+func pushTelemetry(content []byte) {
+	telemetryType := telemetry.TelemetryType("SCC-CONNECT-MANDATORY")
+	class := telemetry.MANDATORY_TELEMETRY
+	tags := telemetry.Tags{"mandatory", "scc", "connect"}
+	flags := telemetry.GENERATE | telemetry.SUBMIT
```

Comment:
Note that the telemetry.SUBMIT flag should only really be used by an application that wants the telemetry to be immediately submitted without relying on the background submission agent (systemd timer triggered) to bundle up and send the telemetry at a later date.

Additionally if you plan to generate multiple telemetry data items, and want to submit immediately, only the last Generate() call needs to include the SUBMIT flag.